package.json

```diff
@@ -5,9 +5,9 @@
   "license": "MIT",
   "scripts": {
     "test": "tsc --noEmit",
-    "eslint": "eslint '**/*.d.ts'",
```

Comment:
do single quotes and double quotes have different functions here?

Reply:
With single quotes, it was throwing no files matching the pattern error. This was fixed when escaped double quotes were used.

Reply:
I have not encountered this problem, what operating system are you using? This may be related to the operating system, I am using ubuntu 18.04 zsh.

Reply:
I'm not sure if this is OS related. I'm using Windows 10.

Reply:
Thank you for your PR, I will test it later.